### PR TITLE
source-client-success contribution from MindNumbing

### PR DESCRIPTION
**Pull request by MindNumbing**
## What

This PR adds a new connector for source Client Success (source-client-success).

Contributor-provided connector description:

Connector for the working Client Success API endpoints
## Details

<details><summary>Expand for list of spec properties and streams</summary>

### Configuration

| Input | Type | Description | Default Value |
|-------|------|-------------|---------------|
| `start_date` | `string` | Start date.  |  |
| `api_key` | `string` | API Key.  |  |

### Streams
| Stream Name | Primary Key | Pagination | Supports Full Sync | Supports Incremental |
|-------------|-------------|------------|---------------------|----------------------|
| Clients | id | DefaultPaginator | ✅ |  ✅  |
| Contacts | id | DefaultPaginator | ✅ |  ✅  |
| Contracts | id | DefaultPaginator | ✅ |  ✅  |
| Products | id | DefaultPaginator | ✅ |  ✅  |
| Custom Fields | fieldId | No pagination | ✅ |  ❌  |

</details>

## Reviewer checklist
- [ ] Run `/format-fix` to resolve any formatting errors
- [ ] Help the contributor upload an icon that follows our icon design guidelines
- [ ] Click into the CI workflows that wait for a maintainer to run them, which should trigger CI runs

### CODE REVIEW

**Comment by MindNumbing** [comment]
👋 First time contributor here. I created this PR using the no-code builder tool. The API docs I followed to create this is [here](https://clientsuccess.readme.io).

Unfortunately many of the API endpoints are down for maintenance so this will need to be iterated on as the API becomes more stable.

**Comment by natikgadzhi** [comment]
@MindNumbing can you pretty please upload an icon file (replace the existing icon.svg) in this PR? Needs to be a square, centered SVG. 

Otherwise, since it's from Builder, even doing a small subset of streams is useful, so once we have the icon in, I'll clean up the metadata and merge!

**Comment by MindNumbing** [comment]
The new changes are switching the auth type from an API_Key to a session_token retrieval flow using the V1 auth endpoint.

**Comment by MindNumbing** [comment]
> @MindNumbing can you pretty please upload an icon file (replace the existing icon.svg) in this PR? Needs to be a square, centered SVG.
> 
> Otherwise, since it's from Builder, even doing a small subset of streams is useful, so once we have the icon in, I'll clean up the metadata and merge!

I will add an icon based on the favicon for Client-Success.

**Comment by MindNumbing** [comment]
@natikgadzhi, what sort of metadata changes are required? I could address any simple issues before handing it off.

**Comment by natikgadzhi** [comment]
allowedHosts key should contain the actual api host.

File: airbyte-integrations/connectors/source-client-success/metadata.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+metadataSpecVersion: "1.0"
+data:
+  allowedHosts:
+    hosts:
+      - "api.clientsuccess.com"
```
Comment (natikgadzhi) [inline airbyte-integrations/connectors/source-client-success/metadata.yaml:5]: Thank you!

**Comment by natikgadzhi** [comment]
@MindNumbing this is just waiting for the CLA agreement (comment above) and the icon.

**Comment by marcosmarxm** [comment]
Nice contribution, @MindNumbing. Unfortunately, we can't merge it without the CLA being signed, and there is an incorrect icon. Please ping me if you want to revisit this contribution. For now, I'm closing it.

**Pull request closed without merging**